_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,141 @@
+<sup>_Yearly profiles of US electricity generation by carbon intensity._</sup>
+
+From the plot we can see that the amount of generation peaked around 2007 and has been roughly stable since then.  While some increases in total emissions occurred after 2001, by looking at 2016, we see that generation from fossil fuels is at the same level as it was in 2001. Furthermore, we can see two horizontal "shelves", with the lower one around 400 kg CO<sub>2</sub>/MWh corresponding to generation from natural gas, with the upper one at 900 kg CO<sub>2</sub>/MWh corresponding to generation from coal[^Not_Life_Cycle_Emissions_Intensity].  In 2016, these shelves are quite visible, and the light gray area represents a large amount of emissions that were reduced by switching from coal to natural gas.  Overall it's clear that the US has been steadily decarbonizing the electricity sector.
+
+Another view is shown in the plot below which examines how much of electricity generation is responsible for how much of total CO<sub>2</sub> emissions.  The motivation for this is that if you find that 90% of CO<sub>2</sub> emissions are from 10% of your electricity generation, then large reductions in CO<sub>2</sub> emissions can be achieved by changing only a small fraction of the existing infrastructure.  This plot uses a similar approach as the previous ones, with the following steps:
+
+* Sort electricity generation per year, from highest CO<sub>2</sub> intensity to lowest.
+* Calculate total generation and CO<sub>2</sub> emissions per year
+* Calculate cumulative generation and CO<sub>2</sub> emissions per year
+* Divide these cumulative totals by the yearly totals to get cumulative percentages
+
+Starting at 2001, this shows that 27% of electricity generation was from nuclear and renewables, with the remaining 73% from fossil fuels.  Over time, more renewables (such as large amounts of installed wind capacity), more efficient power plants, and a switch from coal to natural gas have pushed this curve to the left and steepened the slope.  As of 2016, 75% of CO<sub>2</sub> emissions come from only 35% of electricity generation, with half of CO<sub>2</sub> coming from just 21%.
+
+![Percent Emissions per Percent Generation]({{ site.baseurl }}public/images/YearlyPercentEmissionsPercentGeneration.gif)
+<sup>_Percent of CO2 emissions coming from percent of electricity generation._</sup>
+
+# Conclusions
+
+What has been shown here is just a small subset of what's available in the [open data published by the EIA](https://www.eia.gov/opendata/){:target="_blank"}. Through this post he hope to have shown some approaches that can help make sense of a deluge of raw data, and tell stories that are not necessarily obvious.  At Invenia, these types of exercises are part of our daily jobs.  Electricity grids are complex systems, and understanding them requires having a grasp of interconnected phenomenon spanning multiple geographic and time scales.
```

Comment:
--> "In the above analysis and discussion we looked at only a small subset of what is available in the open data published by the EIA{:target="_blank"}, using a couple of techniques that can help make sense of a deluge of raw data, and tell stories that are not necessarily obvious."

Reply:
"At Invenia, these types of exercises are part of our daily jobs.  Electricity grids are complex systems, and understanding them requires having a grasp of interconnected phenomenon spanning multiple geographic and time scales."

I think delete the Invenia sentence?  Seems too PR like?
For last sentence, I think we can do better, but I don't have a good suggestion atm.